lib/watir/elements/date_field.rb

```diff
@@ -0,0 +1,41 @@
+module Watir
+  class DateField < Input
+
+    #
+    # Enter the provided value.
+    #
+
+    def set!(date)
+      date = Date.parse date if date.is_a?(String)
+
+      message = "DateTimeField##{__callee__} only accepts instances of Date or Time"
```

Comment:
`DateTimeField` should just be `DateField`.

Reply:
I'm winning with the copy/paste today